src/puppetlabs/trapperkeeper/services/scheduler/scheduler_core.clj

```diff
@@ -62,6 +63,46 @@
       ; this can occur if the interface is being used while the scheduler is shutdown
       (log/error e (i18n/trs "Failed to schedule job")))))
 
+(defn interval
+  [^Scheduler scheduler repeat-delay f group-name]
+  (try
+    (let [job-name (Key/createUniqueName group-name)
+          job (build-executable-job f job-name group-name {:interval repeat-delay})
+          schedule (-> (SimpleScheduleBuilder/simpleSchedule)
+                       (.withIntervalInMilliseconds repeat-delay)
+                       (.withMisfireHandlingInstructionNextWithRemainingCount)
+                       (.repeatForever))
+
+          trigger (-> (TriggerBuilder/newTrigger)
+                      (.withSchedule schedule)
+                      (.startNow)
+                      (.build))]
+      (.scheduleJob scheduler job trigger)
+      (.getJobKey trigger))
+    (catch SchedulerException e
+      ; this can occur if the interface is being used while the scheduler is shutdown
+      (log/error e (i18n/trs "Failed to schedule job")))))
+
+(defn interval-after
+  [^Scheduler scheduler initial-delay repeat-delay f group-name]
+  (try
+    (let [job-name (Key/createUniqueName group-name)
+          job (build-executable-job f job-name group-name {:interval repeat-delay})
+          schedule (-> (SimpleScheduleBuilder/simpleSchedule)
+                       (.withIntervalInMilliseconds repeat-delay)
+                       (.withMisfireHandlingInstructionNextWithRemainingCount)
```

Comment:
I don't understand how this relates to the work in the job with rescheduling if skipped? Maybe a comment would be good?

Reply:
Yes.  I'm going to reevaluate this option, along with the in-job logic soon.  I found some additional docs on Quartz that detail the threshold for when this is applied.  By default it is quite large.

Reply:
Sorry, didn't realize this was already merged. That sounds good.